pwnlib/shellcraft/templates/thumb/mov.asm

```diff
@@ -18,93 +18,60 @@
 
 Example:
 
-    >>> print shellcraft.thumb.mov('r1','r2').rstrip()
-        mov r1, r2
-    >>> print shellcraft.thumb.mov('r1', 0).rstrip()
+   >>> print shellcraft.thumb.mov('r1','r2').rstrip()
+       mov r1, r2
+   >>> print shellcraft.thumb.mov('r1', 0).rstrip()
+       eor r1, r1
+   >>> print shellcraft.thumb.mov('r1', 10).rstrip()
```

Comment:
This is more instructions than the old version, which did not emit NUL or newline (afaict)